wandb/sdk/artifacts/artifact_manifest_entry.py

```diff
@@ -114,14 +114,14 @@ def ref_target(self) -> Union[FilePathStr, URIStr]:
 
         Raises:
             ValueError: If this artifact entry was not a reference.
+            FileNotFoundError: If the file does not exist at the reference URI.
```

Comment:
`local=False` means we just return the URL without talking to S3, so this will never be raised here.